lib/game.rb

```diff
@@ -23,6 +23,7 @@ def start
     while play
         puts @board.print_visual
         take_turn
+        win?
```

Comment:
Love the addition of the "win? method" which checks for any of the possible "Connect 4" win conditions